packages/notifi-react-card/lib/components/NotifiEmailInput.tsx

```diff
@@ -58,6 +58,7 @@ export const NotifiEmailInput: React.FC<NotifiEmailInputProps> = ({
   return (
     <>
       {intercomView ? (
+        email &&
```

Comment:
why is this necessary?

Reply:
I was thinking of if there is no email address showing here, we don't need to show the verification link, hmm, we could probably just use the emailIdThatNeedsConfirmation to check it. Let me remove the email here.